how-to/create-snapshot.md

```diff
@@ -10,16 +10,31 @@ date: 1/17/2020
 # Create Snapshot
 
 A snapshot in the context of SysKit Trace is a collection of all the Office 365 configuration settings taken at a specific point in time. 
+There are two ways to obtain a snapshot in SysKit Trace:
```

Comment:
Generate/Create a snapshot?

Reply:
you are correct, it makes much more sense. replaced it with create.